tracing-syslog/src/syslog.rs

```diff
@@ -0,0 +1,220 @@
+use std::{borrow::Cow, ffi::CStr};
+use tracing_core::{
+    field::{Field, Visit},
+    Collect, Event, Level,
+};
+use tracing_subscriber::{
+    registry::{LookupSpan, Registry},
+    subscribe::{CollectExt, Context, Subscribe},
+};
+
+/// `syslog` options.
+#[derive(Copy, Clone, Debug, Default)]
+pub struct Options(libc::c_int);
+
+impl Options {
+    /// Log the pid with each message.
+    pub const LOG_PID: Self = Self(libc::LOG_PID);
+    /// Log on the console if errors in sending.
+    pub const LOG_CONS: Self = Self(libc::LOG_CONS);
+    /// Delay open until first syslog() (default).
+    pub const LOG_ODELAY: Self = Self(libc::LOG_ODELAY);
+    /// Don't delay open.
+    pub const LOG_NDELAY: Self = Self(libc::LOG_NDELAY);
+    /// Don't wait for console forks: DEPRECATED.
+    pub const LOG_NOWAIT: Self = Self(libc::LOG_NOWAIT);
+    /// Log to stderr as well.
+    pub const LOG_PERROR: Self = Self(libc::LOG_PERROR);
+}
+
+impl std::ops::BitOr for Options {
+    type Output = Self;
+    fn bitor(self, rhs: Self) -> Self::Output {
+        Self(self.0 | rhs.0)
+    }
+}
```

Comment:
Hmm, not sure if this carries its weight. Why do you think this is valuable?

Reply:
So you can set multiple options at the same time:
```rust
use tracing_syslog::Options;
// Log PID with messages and log to stderr as well as `syslog`.
let opts = Options::LOG_PID | Options::LOG_PERROR;
```
(added this example to `Option`'s docs)